net/pfSense-pkg-Quagga_OSPF/files/usr/local/pkg/quagga_ospfd.inc

```diff
@@ -291,11 +291,10 @@ function quagga_ospfd_install_conf() {
 
 	/* Make bgpd config, add password and logging */
 	if (isset($config['installedpackages']['quaggaospfdraw']['config'][0]['bgpd'])
-	    && !empty($config['installedpackages']['quaggaospfdraw']['config'][0]['bgpd'])) {
+		&& !empty($config['installedpackages']['quaggaospfdraw']['config'][0]['bgpd'])) {
```

Comment:
These should be [indented with four spaces](https://www.netgate.com/docs/pfsense/development/developer-style-guide.html#indent-style)